README.md

```diff
@@ -1,123 +1,73 @@
-   $ mv env.example .env
-   ```
-   **Note** Some local environments require Go projects are run from the `$HOME/go/src` directory. If any `module not found` errors are reported, try moving it.
-
-   Also be sure to fill in the environment variables before running a command.
-
-3. Run commands
+## Installation
+To get started with `gwa`, follow the detailed instructions provided in the DevHub [how-to](https://dev.developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/how-to/gwa-install/) guide. This guide covers all the steps required to install and configure the tool on your system.
 
-   Test any commmands by running `just run` in the `cwd`. You can also use `$ just test` to run all tests.
+---
 
-   ```sh
-   $ just run gateway current
-   your-gateway
-   $ just test
-   ?   	github.com/bcgov/gwa-cli	[no test files]
-   ok  	github.com/bcgov/gwa-cli/cmd
-   ok  	github.com/bcgov/gwa-cli/pkg
-   ```
+## Usage
 
-4. Set up your IDE
+### Use `--help`
 
-   Go has great tooling which is required to ensure code contributed is formatted consitently and is type-safe.
+To make the most out of the `gwa` tool, you can use the `--help` command to get detailed information on its various commands and options. Here's how you can use it:
 
-   - **VSCode:** Install the [official Go extension](https://marketplace.visualstudio.com/items?itemName=golang.Go)
-   - **NeoVim:** [go.nvim](https://github.com/ray-x/go.nvim) is a great plugin
+#### Basic Usage of `--help`
+The `--help` flag can be used with any command within the `gwa` tool to display information about that command, including its purpose, options, and examples.
 
-#### Nix develop (optional)
+1. **General Help**: To get an overview of all available commands, simply run:
+    ```sh
+       gwa --help
+    ```
+    This will list all the primary commands and provide a brief description of each.
+       
+2. **Command-Specific Help**: To get detailed information about a specific command, use `--help` with the command in question. For example, if you want to learn more about the gateway command, you can run:
+    ```sh
+       gwa gateway --help
+    ```
+    This will provide detailed information about the gateway command, including its syntax, options, and examples.
 
-Instead of installing Go on your system you can alternatively use Nix to manage Go and other dependencies for you. This option assumes you already running NixOS or have Nix installed.
+### Explore CLI commands
 
-Simply run `$ nix develop` in the project root. This will pull all the dependencies needed to work on this project. Remember to run `exit` to leave the Nix shell when done.
+Learn how to effectively use `gwa` by exploring the available CLI commands and their options. Refer to the comprehensive DevHub [CLI commands page](https://dev.developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/reference/gwa-commands/) for detailed instructions and examples on how to utilize the tool for your needs.
```

Comment:
```suggestion
Explore `gwa` commands in an easy-to-read reference format on the DevHub 
[CLI commands page](https://developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/reference/gwa-commands/).
The content on this page is the same as the `--help` output.
```
Shorted but also note we should generally point to `developer...` not `dev.developer`.